### PR TITLE
Charm 7.0.1

### PR DESCRIPTION
**Pull request by ericjbohm**
Change License to Apache 2.0 with LLVM exceptions.
Otherwise unchanged from 7.0.0

### CODE REVIEW

File: CHANGES [hunk 1]
```diff
@@ -1,6 +1,20 @@
 This file describes the most significant changes. For more detail, use
 'git log' on a clone of the charm repository.
 
+===============================================================================
+What's new in Charm++ 7.0.1
+================================================================================
+
+This is a change license only release, with the following change:
```
Comment (matthiasdiener) [inline CHANGES:8]: maybe:
```suggestion
This is a license-only release, with the following change:
```

**Comment by ericjbohm** [comment]
> Not sure how to set this up as a PR, merging onto main looks like a lot of changes, but this looks like we're trying to make a change to 7.0.

For that matter, not sure if needs to be a PR, since it already exists as a branch in the repo, so anyone can just check it out and use it as normal.

**Comment by matthiasdiener** [comment]
> Not sure how to set this up as a PR, merging onto main looks like a lot of changes, but this looks like we're trying to make a change to 7.0.

Maybe make the version number changes in this PR on top of main, then create a new branch `charm-7.0.1` off of `charm-7.0.0` and just cherry pick the version number commit into charm-7.0.1`.

**Comment by ericjbohm** [comment]
That is how I created the 7.0.1 branch.

**Comment by ericjbohm** [comment]
no PR required.

**Pull request closed without merging**